src/cocotb_tools/runner.py

```diff
@@ -163,6 +170,7 @@ def build(
             hdl_library: The library name to compile into.
             verilog_sources: Verilog source files to build.
             vhdl_sources: VHDL source files to build.
+            sources: Language-agnostic list of source files to build.
```

Comment:
I think we should add some documentation about the build order for `sources`, `vhdl_sources`, and `verilog_sources`.